scripts/coverage.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
```

Comment:
This is the same as `set -e`

So you could also do: `set -eo pipefail` (combine some lines)

Or if you just wanted to leave it as is but combine lines, you could do:

`set -o errexit -o pipefail`

Or you could just ignore me altogether. :wink: